ovnimp.go

```diff
@@ -188,11 +193,23 @@ func (odbi *ovndb) execute(cmds ...*OvnCommand) error {
 		}
 	}
 
-	_, err := odbi.transact(odbi.db, ops...)
-	if err != nil {
-		return err
+	results, err := odbi.transact(odbi.db, ops...)
+	if err == nil {
```

Comment:
it would be better to:
if  err != nil {
    return nil, err
}

to avoid the indent of the big block.